crt_portal/cts_forms/forms.py

```diff
@@ -755,16 +755,15 @@ def __init__(self, *args, **kwargs):
         ModelForm.__init__(self, *args, **kwargs)
         Contact.__init__(self, *args, **kwargs)
         # CRT views only
-        self.fields['intake_format'] = TypedChoiceField(
+        self.fields['intake_format'] = ChoiceField(
             choices=(
-                EMPTY_CHOICE,
                 ('letter', 'letter'),
                 ('phone', 'phone'),
                 ('fax', 'fax'),
                 ('email', 'email'),
             ),
-            widget=Select(attrs={
-                'class': 'usa-select mobile-lg:grid-col-7',
+            widget=UsaRadioSelect(attrs={
+                'name': 'reported_reason',
```

Comment:
@hakhalid11, can you say more about this change? I'm not sure what the resulting effect is on the front end.

Reply:
Your right there is no need for the widget attr looks like something that I should have removed. Thanks for pointing it 👍